Sources/Extensions/SwiftStdlib/StringExtensions.swift

```diff
@@ -229,7 +229,7 @@ public extension String {
 	///		"Hello world!".length -> 12
 	///
 	public var length: Int {
```

Comment:
We should probably deprecate this one. It's not even a good name because it can easily be confused with `NSString.length`

Reply:
💯, also `String.count` can be used in Swift 4